python/export_util.py

```diff
@@ -76,16 +77,82 @@ def convert_to_isoformat(
         return property
 
 
-def get_graph(ctx: mgp.ProcCtx) -> List[Union[Node, Relationship]]:
+def to_duration_iso_format(value: timedelta) -> str:
```

Comment:
To be consistent with Python’s `datetime.isoformat` method:
```suggestion
def to_duration_isoformat(value: timedelta) -> str:
```